rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -0,0 +1,197 @@
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-15; does not follow typical OpenShift documentation formatting
+//---
+//date: '2023-07-27'
+//title: Prerequisites Checklist to Deploy ROSA Cluster with STS 
+//tags: ["ROSA", "STS"]
+//authors:
+//  - Byron Miller
+//  - Connor Wooley
+//  - Diana Sari
+//---
+
+This is a checklist of prerequisites needed to create a {product-title} (ROSA) classic cluster with link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[STS]. 
+
+[NOTE] 
+==== 
+This is a high level checklist and your implementation can vary. 
+====
+
+Before running the installation process, verify that you deploy this from a machine that has access to:
+
+* The API services for the cloud to which you provision.
+* Access to `api.openshift.com` and `sso.redhat.com`. 
+* The hosts on the network that you provision.
+* The internet to obtain installation media.
+
+== Accounts and CLIs Prerequisites
+
+Accounts and CLIs you must install to deploy the cluster.
+
+=== AWS account
+
+* Gather the following details:
+** AWS IAM User
+** AWS Access Key ID
+** AWS Secret Access Key
+* Ensure that you have the right permissions as detailed link:https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol.html[AWS managed IAM policies for ROSA] and xref:../rosa_architecture/rosa-sts-about-iam-resources.html[About IAM resources for ROSA clusters that use STS].
+* See xref:../rosa_planning/rosa-sts-aws-prereqs.html#rosa-account_rosa-sts-aws-prereqs[Account] for more details. 
+
+=== AWS CLI (`aws`)
+
+* Install from https://aws.amazon.com/cli/[AWS Command Line Interface] if you have not already.
+* Configure the CLI:
++
+. Enter `aws configure` in the terminal: 
++
+[source,terminal]
+----
+$ aws configure
+----
++
+. Enter the AWS Access Key ID and press *enter*.
+. Enter the AWS Secret Access Key and press *enter*.
+. Enter the default region you want to deploy into.
+. Enter the output format you want, “table” or “json”. 
+. Verify the output by running:
++
+[source,terminal]
+----
+ $ aws sts get-caller-identity
+----
++
+. Ensure that the service role for ELB already exists by running:
++
+[source,terminal]
+----
+$ aws iam get-role --role-name "AWSServiceRoleForElasticLoadBalancing"
+----
++
+.. If it does not exist, run:
++
+[source,terminal]
+----
+$ aws iam create-service-linked-role --aws-service-name "elasticloadbalancing.amazonaws.com"
+----
+
+=== Red Hat account
+
+* Create a https://console.redhat.com/[{hybrid-console}]  account if you have not already.
+
+=== ROSA CLI (`rosa`)
+
+. Enable ROSA from your AWS account on the https://console.aws.amazon.com/rosa/[AWS console] if you have not already.
+. Install the CLI from xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html[Installing the Red Hat OpenShift Service on AWS (ROSA) CLI, rosa[] or from the OpenShift console https://console.redhat.com/openshift/downloads#tool-rosa[AWS console].
```

Comment:
Renders as 
![image](https://github.com/openshift/openshift-docs/assets/16167833/37a8a8f1-c83b-4669-a399-49dbd8691eff)


Also, I'm not sure how line 91 works as a link. It's just a URL attached to the hyperlinked text, but I thought you needed `link:`.